profile/README.md

```diff
@@ -37,13 +34,9 @@
 
 ## What do we NOT do?
 
-* We don't coach people to crack GSoC. That is not our main objective as a society. If you want to join KOSS because of this sole reason, you are better suited to ask previous GSoCers about the same.
+* We don't coach people to crack GSoC.
```

Comment:
We can keep it a little generic and not specific to GSoC, as it is assumption reached upon only by a few people in our college and not outside of it. 

```suggestion
* We don't train people to crack any coding competitions or hackathons.
```